src/main/java/de/komoot/photon/nominatim/NominatimConnector.java

```diff
@@ -353,7 +357,8 @@ public void processRow(ResultSet rs) throws SQLException {
 		});
 
 
-		template.query("SELECT place_id, osm_id, parent_place_id, startnumber, endnumber, interpolationtype, postcode, country_code, linegeo FROM location_property_osmline ORDER BY geometry_sector; ", new RowCallbackHandler() {
+		template.query("SELECT place_id, osm_id, parent_place_id, startnumber, endnumber, interpolationtype, postcode, country_code, linegeo FROM location_property_osmline " +
+				countryCodeStr + " ORDER BY geometry_sector; ", new RowCallbackHandler() {
```

Comment:
This looks wrong. It should be 'WHERE country_code ...' not 'AND country_code ...'.